code/game/objects/structures/tables_racks.dm

```diff
@@ -545,6 +554,72 @@
 /obj/structure/table/reinforced/brass/ratvar_act()
 	obj_integrity = max_integrity
 
+/obj/structure/table/tray
+	name = "surgical tray"
+	desc = "A small metal tray with wheels."
+	anchored = 0
+	smooth = SMOOTH_FALSE
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "tray"
+	buildstack = /obj/item/stack/sheet/mineral/titanium
+	buildstackamount = 2
+	var/list/typecache_can_hold = list(/mob, /obj/item)
+	var/list/held_items = list()

```

Comment:
I was about to tell you to `Cut` this, but it seems you used `UID` so this should be good. Yay.